examples/step-79/doc/intro.dox

```diff
@@ -6,23 +6,23 @@ structure that is bearing some load. Ideally, we would like to minimize the
 maximum stress placed on a structure by selecting a region $E$ where material is
 placed. In other words,
 @f[
-  \text{Minimize}\norm{\boldsymbol{\sigma} (\textbf{u}) }_\infty
+  \text{Minimize}\norm{\boldsymbol{\sigma} (\mathbf{u}) }_\infty
 @f]
 @f[
   \text{subject to } |E|\leq V_{\max},
 @f]
 @f[
-  \text{and } \nabla \cdot \boldsymbol{\sigma} +\textbf{ F} = \mathbf{0}.
+  \text{and } \nabla \cdot \boldsymbol{\sigma} + \mathbf{F} = \mathbf{0}.
 @f]
-Here, $\sigma=C\varepsilon(\mathbf u)$ is the stress within the body that is
+Here, $\boldsymbol{\sigma} = \mathbf{C} : \boldsymbol{\varepsilon}(\mathbf{u})$ is the stress within the body that is
```

Comment:
\sigma, C and \varepsilon are all tensor quantities, so I formatted them accordingly.

Reply:
I also added the `:` double contraction symbol where appropriate.